### PR TITLE
Remove Why3 generator

### LINKED ISSUE #2195: Remove Why3 translator
The Why3 output is not maintained and there doesn't seem to be a plan for maintaining it. Would it make sense removing it?

### PR DESCRIPTION
**Pull request by axic**
Fixes #2195.

### CODE REVIEW

File: solc/CommandLineInterface.cpp [hunk 1]
```diff
@@ -613,8 +598,7 @@ Allowed options)",
 		(g_argSignatureHashes.c_str(), "Function signature hashes of the contracts.")
 		(g_argNatspecUser.c_str(), "Natspec user documentation of all contracts.")
 		(g_argNatspecDev.c_str(), "Natspec developer documentation of all contracts.")
-		(g_argMetadata.c_str(), "Combined Metadata JSON whose Swarm hash is stored on-chain.")
-		(g_argFormal.c_str(), "Translated source suitable for formal analysis.");
```
Comment (axic) [inline solc/CommandLineInterface.cpp]: Should we keep this option to emit a message "Why3 support has been removed." for the moment?

Comment (chriseth) [reply solc/CommandLineInterface.cpp]: Yes, I think that's a good idea.

Comment (axic) [reply solc/CommandLineInterface.cpp]: Added.

**Comment by axic** [comment]
Need to remove tests too.

**Comment by axic** [comment]
@chriseth this is ready

**Review by chriseth — APPROVED** [review]

**Pull request merged**